recipes/conversation_assistant/conversation.js

```diff
@@ -55,26 +58,26 @@ const tjConfig = {
 // };
 
 async function converse(message) {
-    let assistantSessionId;
 
     // set up the session if needed
     if (!assistantSessionId) {
+        console.log("no session id detected");
         try {
-            console.log(`creating assistant session, sessionId: ${config.assistantId}`);
+            console.log(`creating assistant session, sessionId: ${config.environmentId}`);
             const body = await assistant.createSession({
-                assistantId: config.assistantId,
+                assistantId: config.environmentId,
             });
             console.log(`response from _assistant.createSession(): ${body.result}`);
             assistantSessionId = body.result.session_id;
         } catch (err) {
-            console.error(`error creating session for ${TJBot.SERVICES.ASSISTANT} service. please check that tj.configuration.converse.assistantId is defined.`);
+            console.error(`error creating session for Assistant service. please check that tj.configuration.converse.environmentId is defined.`);
```

Comment:
`tj.configuration.converse.environmentId` is a bit confusing. maybe we should say, "please check that envrionmentId has been defined in config.js" or something like that?

Reply:
Agreed!